projects/skia/skdescriptor_deserialize.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 1024
```

Comment:
This is discouraged, please prefer using an explicit size check in the fuzz target source  code (i.e. add `if (size > 1024) return 0;` in the beginning of `LLVMFuzzerTestOneInput`). Not a blocking issue, can be addressed later

Reply:
Sounds good - I was unaware of that policy change.